docs/root/intro/version_history.rst

```diff
@@ -1,22 +1,24 @@
 Version history
 ---------------
 
-1.7.0
-=====
+1.7.0 (Pending)
+===============
 
 * Added :ref:`weighted round robin
   <arch_overview_load_balancing_types_round_robin>` support. The round robin
   scheduler now respects endpoint weights and also has improved fidelity across
   picks.
-* admin: added :ref:`/config_dump endpoint <operations_admin_interface_config_dump>` for dumping current configs
+* access log: ability to format START_TIME
+* admin: added :http:get:`/config_dump` for dumping current configs
+* admin: added :http:get:`/stats/prometheus` as an alternative endpoint for getting stats in prometheus format.
 * admin: added :ref:`/runtime_modify endpoint <operations_admin_interface_runtime_modify>` to add or change runtime values
+* admin: mutations must be sent as POSTs, rather than GETs. Mutations include:
+  :http:post:`/cpuprofiler`, :http:post:`/healthcheck/fail`, :http:post:`/healthcheck/ok`,
+  :http:post:`/logging`, :http:post:`/quitquitquit`, :http:post:`/reset_counters`,
+  :http:post:`/runtime_modify?key1=value1&key2=value2&keyN=valueN`,
 * admin: removed `/routes` endpoint; route configs can now be found at the :ref:`/config_dump endpoint <operations_admin_interface_config_dump>`.
-* access log: ability to format START_TIME
-* admin: added ``/stats/prometheus`` as an alternative endpoint for getting stats in prometheus format.
 * cli: added --config-yaml flag to the Envoy binary. When set its value is interpreted as a yaml
   representation of the bootstrap config and overrides --config-path.
-* health check: added support for EDS delivered :ref:`endpoint health status
```

Comment:
I think this shouldn't be deleted?

Reply:
ouch, sorry.  I don't know what happened.  Good catch.  I did more sorting though, as the categories were sorted but not the entries within each category.

Reply:
actually I suspect that the format-fixer elided them...another block disappeared on me after running that.  Or I could've fat-fingered it...I will investigate a little more.